src/pages/home/report/ReportActionItem.tsx

```diff
@@ -105,6 +106,9 @@ type ReportActionItemOnyxProps = {
 
     /** Transaction associated with this report, if any */
     transaction: OnyxEntry<OnyxTypes.Transaction>;
+
+    /** The transaction (linked with the report action) route error */
+    linkedTransactionRouteError: OnyxEntry<Errors>;
```

Comment:
@bernhardoj Sorry I missed this. Why are taking the error object `OnyxEntry<Errors>` alone? Can't we have the linkedTransaction itself as a prop and take the error inside the class? Similar to how `transaction` object is present.

Reply:
We just care about the error object, so @tgolen [suggested to use selector](https://github.com/Expensify/App/issues/38213#issuecomment-2088637569) to optimize it.